packages/permissions/src/utils/has-permissions.ts

```diff
@@ -255,6 +256,13 @@ const getDataFenceByTypeAndGroup = (
   return null;
 };
 
+const getIsPermissionOverwritingDataFence = (
+  actualPermissions: TPermissions | null,
+  actualDataFence: TActualDataFence
+) => {
+  return false;
```

Comment:
hmmmm. this is always returning false 🤔

Reply:
nevermind this I was reviewing commit by commit 😓

Reply:
This is my story: bolt something in, make it not change anything. Adjust behaviour :)